frontend/app_flowy/packages/flowy_infra/lib/theme.dart

```diff
@@ -79,49 +88,55 @@ class AppTheme {
           ..tint8 = const Color(0xffdefff1)
           ..tint9 = const Color(0xffdefff1)
           ..main1 = const Color(0xff00bcf0)
-          ..main2 = const Color(0xff00b7ea);
+          ..main2 = const Color(0xff00b7ea)
+          ..textColor = _black;
 
       case ThemeType.dark:
         return AppTheme(isDark: true)
-          ..surface = const Color(0xff252525)
-          ..hover = const Color(0xFFe0f8ff) //
-          ..selector = const Color(0xfff2fcff)
+          ..surface = _black
+          ..hover = _white //
+          ..selector = _black //support icon
           ..red = const Color(0xfffb006d)
           ..yellow = const Color(0xffffd667)
           ..green = const Color(0xff66cf80)
-          ..shader1 = const Color(0xff333333)
-          ..shader2 = const Color(0xff4f4f4f)
+          ..shader1 = _white //text and icons
```

Comment:
The shader1, shader 2... are the AppFlowy theme colors. It should have dark mode colors. DM you the details